<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContornoEscolha</class>
 <widget class="QMainWindow" name="ContornoEscolha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibração Automatica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="COMBO">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="PROGRESS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LABEL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BOTAO">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Calibração</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Objetos Encontrados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>175</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cores:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="COMBO_COR">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Amarelo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ciano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laranja</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vermelho</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rosa</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="SALVAR">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrado</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BOTAO_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Finalizar Calibração</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
